lib/core.js

```diff
@@ -53,6 +53,32 @@ impress.applicationConfigDefinition = api.definition.require(
   'config.application.definition'
 );
 
+
+impress.loadSchemas = callback => {
+  const serverConfigSchemas = api.path.join(
+    impress.dir, 'schemas', 'config.impress.definition'
+  );
+
+  const applicationConfigSchemas = api.path.join(
+    impress.dir, 'schemas', 'config.application.definition'
+  );
+
+  api.metaschema.fs.loadAndCreate(
+    [serverConfigSchemas, applicationConfigSchemas],
+    api,
+    (err, metaschema) => {
+      if (err) {
+        console.error(err);
+        return;
+      }
+
+      api.metaschema = metaschema;
```

Comment:
I am not sure this is the right decision to reassign `api.metaschema`. Maybe it is better to name it `api.ms`?